### PR TITLE
Add New Port Spacewing War

### PR DESCRIPTION
**Pull request by MadShmupper**
# New Port for { Spacewing War }

## Game Information
- **Title**: { Spacewing War }
- **URL**: { https://store.steampowered.com/app/1979470/Spacewing_War/ }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by kloptops — APPROVED** [review]
👍

**Comment by MadShmupper** [comment]
Kloptops & Cebion, I goofed up and pr'd this one before I had all resolutions by mistake. While I've no reason to believe it won't work on higher res, I'm still awaiting a high res test. Sorry for the hiccup fellas. 👾🤖👽

**Comment by MadShmupper** [comment]
Okay, Cebion.. This one is officially ready now. All tests in🙂

File: ports/spacewingwar/gameinfo.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<gameList>
+  <game>
+    <path>./Spacewing War.sh</path>
+    <name>Spacewing War</name>
+    <desc>Blast through adorable enemies in a side scrolling blast that looks from the past. </desc>
+    <releasedate>20220608T000000</releasedate>
+    <developer>Pneuma Games</developer>
+    <publisher>Eastasiasoft Limited</publisher>
+    <genre>shmup</genre>
+    <image>./spacewing War/cover.png</image>
```
Comment (Cebion) [inline ports/spacewingwar/gameinfo.xml:11]: foldername wrong

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Slayer366 — CHANGES_REQUESTED** [review]

**Review by Slayer366 — APPROVED** [review]
Lookin' good!

**Comment by MadShmupper** [comment]
> Lookin' good!

Thank ya kindly for the help👾

File: ports/spacewingwar/gameinfo.xml (hunk 1 above)
Comment (MadShmupper) [reply ports/spacewingwar/gameinfo.xml]: Fixed🙂

File: ports/spacewingwar/Spacewing War.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+# We log the execution of the script into log.txt
+exec > >(tee "$GAMEDIR/log.txt") 2>&1
+
+cd $GAMEDIR
+
+if [ -f "${controlfolder}/libgl_${CFW_NAME}.txt" ]; then 
+  source "${controlfolder}/libgl_${CFW_NAME}.txt"
+else
+  source "${controlfolder}/libgl_default.txt"
+fi
```
[the remark this answers is not in the document]
Comment (MadShmupper) [reply ports/spacewingwar/Spacewing War.sh]: Fixed this.

**Review by Cebion — APPROVED** [review]

**Comment by MadShmupper** [comment]
😎

**Pull request merged**